released/packages/coq-coqeal/coq-coqeal.dev/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "CoqEAL - The Coq Effective Algebra Library"
+description: """\
+This libary contains a subset of the work that was developed in the context of the ForMath european project (2009-2013). It has two parts:
+- theory (module CoqEAL_theory), which contains formal developments in algebra and optimized algorithms on mathcomp data structures.
+- refinements (module CoqEAL_refinements), which is a framework to ease change of data representation during a proof."""
+maintainer: "Cyril Cohen <cyril.cohen@inria.fr>"
+authors: [
+  "Guillaume Cano"
+  "Cyril Cohen"
+  "Maxime Dénès"
+  "Anders Mörtberg"
+  "Damien Rouhling"
+  "Vincent Siles"
+]
+license: "MIT"
+tags: [
+  "category:Computer Science/Decision Procedures and Certified Algorithms/Correctness proofs of algorithms"
+  "keyword:effective algebra"
+  "keyword:elementary divisor rings"
+  "keyword:Smith normal form"
+  "keyword:mathematical components"
+  "keyword:Bareiss"
+  "keyword:Karatsuba"
```

Comment:
Just in case you resubmit it, please use `Karatsuba multiplication` here, so as to avoid duplicating keywords. (See https://coq.inria.fr/opam/www/)

Reply:
Yes, that was already fixed by @proux01 thanks for raising the issue in the first place.